generators/entity-client/templates/angular/src/main/webapp/app/entities/list/entity-management.component.ts.ejs

```diff
@@ -16,6 +16,12 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
+<%_
+  const notSortableFieldsAfterSearch = !searchEngine ? '' : fields
+      .filter(field => !field.hidden && !field.fieldTypeBoolean && !field.fieldTypeNumeric && !field.fieldTypeTemporal)
```

Comment:
```suggestion
      .filter(field => !field.hidden && !field.sortDisabledForSearchEngine)
```

This logic can be reused at other clients.
- put `sortDisabledForSearchEngine: field.fieldType !== BOOLEAN && !fieldTypeNumeric && !fieldTypeTemporal,`, naming can change.
- extract fieldTypeTemporal and fieldTypeNumeric.

at https://github.com/jhipster/generator-jhipster/blob/5d1410611fa380695038ed9a15dd82cc953546d2/utils/field.js#L226